### PR TITLE
Head english

### PR DESCRIPTION
**Pull request by ericanaughton**
:tea: 

ready to go

### CODE REVIEW

**Comment by nwerbeckes-gpsw** [comment]
:+1:

File: elements/head.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<article class="article">
+  <header class="article-header">
+    <h2><code>&lt;head></code></h2>
```
Comment (rylnd) [inline elements/head.html]: It is very strange that you are only escaping the opening bracket and not the closing bracket in all of these code blocks.

While I trust that it works in whatever browser you are developing against, I would not rely on that behavior across all browsers and all versions, and opt instead to explicitly encode the closing bracket as well:

``` html
<h2><code>&lt;head&gt;</code></h2>
```

Comment (ericanaughton) [inline elements/head.html]: good point! ill make note to get those changed.

**Comment by ericanaughton** [comment]
do i need to make a new PR? or is this ok?

**Comment by rylnd** [comment]
Addressing my comment in this branch/PR is perfectly fine.

**Comment by rylnd** [comment]
I assume that closing/reopening the PR was accidental?

:+1:

**Comment by ericanaughton** [comment]
i closed it, then immediately realized i should ask first, so i re-opened
it :) awesome, thanks

[image: photo]
_Erica Naughton_
Designer/Developer, Adorable.io
+1 866 440 4593 | http://adorable.io | 123 E. Main St - 2nd Floor, Madison
WI 53715
http://twitter.com/adorableio  http://facebook.com/adorableio
✔ Get a Signature like this: Click Here!
http://ws-promos.appspot.com/r?rdata=eyJydXJsIjogImh0dHA6Ly93d3cud2lzZXN0YW1wLmNvbS9lbWFpbC1pbnN0YWxsP3dzX25jaWQ9NjcyMjk0MTIwJnV0bV9zb3VyY2U9ZXh0ZW5zaW9uJnV0bV9tZWRpdW09ZW1haWwmdXRtX2NhbXBhaWduPXByb21vXzU3MTM1NzMyNTA1OTY4NjQiLCAiZSI6ICI1NzEzNTczMjUwNTk2ODY0In0=

**Comment by nwerbeckes-gpsw** [comment]
:+1:

**Pull request merged**